### PR TITLE
small update to link to the rpm

### PR DESCRIPTION
**Pull request by patduin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ We also use Waggle Dance to apply a simple security layer to cloud based platfor
...
+[![Maven Central](https://maven-badges.herokuapp.com/maven-central/com.hotels/waggle-dance-rpm/badge.svg?subject=com.hotels:waggle-dance-rpm)](https://maven-badges.herokuapp.com/maven-central/com.hotels/waggle-dance-rpm) [![Build Status](https://travis-ci.org/HotelsDotCom/waggle-dance.svg?branch=master)](https://travis-ci.org/HotelsDotCom/waggle-dance) [![Coverage Status](https://coveralls.io/repos/github/HotelsDotCom/waggle-dance/badge.svg?branch=master)](https://coveralls.io/github/HotelsDotCom/waggle-dance?branch=master) ![GitHub license](https://img.shields.io/github/license/HotelsDotCom/waggle-dance.svg)
```
Comment (massdosage) [inline README.md]: Shouldn't we also link to the TGZ?

Comment (patduin) [reply README.md]: right I forgot about that one, yeah can do we can add the link. rpm and tgz both enough?

File: README.md [hunk 2]
```diff
@@ -11,7 +11,8 @@ We also use Waggle Dance to apply a simple security layer to cloud based platfor
 
 You can obtain Waggle Dance from Maven Central:
 
-[![Maven Central](https://maven-badges.herokuapp.com/maven-central/com.hotels/waggle-dance/badge.svg?subject=com.hotels:waggle-dance)](https://maven-badges.herokuapp.com/maven-central/com.hotels/waggle-dance) [![Build Status](https://travis-ci.org/HotelsDotCom/waggle-dance.svg?branch=master)](https://travis-ci.org/HotelsDotCom/waggle-dance) [![Coverage Status](https://coveralls.io/repos/github/HotelsDotCom/waggle-dance/badge.svg?branch=master)](https://coveralls.io/github/HotelsDotCom/waggle-dance?branch=master) ![GitHub license](https://img.shields.io/github/license/HotelsDotCom/waggle-dance.svg)
+[![Maven Central](https://maven-badges.herokuapp.com/maven-central/com.hotels/waggle-dance/badge.svg?subject=com.hotels:waggle-dance-bin.tgz)](https://maven-badges.herokuapp.com/maven-central/com.hotels/waggle-dance)
```
Comment (ddcprg) [inline README.md]: in this line the line points at `waggle-dance-bin.tgz`, can we do the same we the rpm?

File: README.md [hunk 3]
```diff
@@ -11,7 +11,8 @@ We also use Waggle Dance to apply a simple security layer to cloud based platfor
...
+[![Maven Central](https://maven-badges.herokuapp.com/maven-central/com.hotels/waggle-dance-rpm/badge.svg?subject=com.hotels:waggle-dance-rpm)](https://maven-badges.herokuapp.com/maven-central/com.hotels/waggle-dance-rpm)  [![Build Status](https://travis-ci.org/HotelsDotCom/waggle-dance.svg?branch=master)](https://travis-ci.org/HotelsDotCom/waggle-dance) [![Coverage Status](https://coveralls.io/repos/github/HotelsDotCom/waggle-dance/badge.svg?branch=master)](https://coveralls.io/github/HotelsDotCom/waggle-dance?branch=master) ![GitHub license](https://img.shields.io/github/license/HotelsDotCom/waggle-dance.svg)
```
Comment (ddcprg) [inline README.md]: we should replace the text `Maven Central` with `Maven Central RPM`

Comment (patduin) [reply README.md]: sure I don't actually see this text displayed though, perhaps when the badge isn't available

File: README.md (hunk 2 above)
Comment (patduin) [reply README.md]: no clue what you mean here. rename waggle-dance-rpm to waggle-dance.rpm??
we're not actually linking to files I just wanted it to be a bit more clear what you can find when you click on the badge. it goes to mvn search, from which you have to click to get the actual files.

**Pull request merged**